### PR TITLE
Fix status code and add private params method

### PR DESCRIPTION
**Pull request by AlexRS90**
## For this PR I

✅  Updated show action using params as a private method
✅  Added status codes to the API answer

### CODE REVIEW

File: app/models/appointment.rb [hunk 1]
```diff
@@ -3,4 +3,5 @@ class Appointment < ApplicationRecord
   validates :city, presence: true
 
   belongs_to :user
+  belongs_to :doctor
```
Comment (codepantha) [inline app/models/appointment.rb:6]: Hi @AlexRS90👋 
Your changes are almost good to go. 
@AlexRS90 please add the has_many through appointments 
```ruby 
has_many :doctors, through: :appointments
``` 
relationship from the users to the doctors table 🙏 
Happy coding 🎉 💻

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @AlexRS90 @micheaol 👋, 

# STATUS: CHANGES REQUIRED ♻️ 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: app/models/appointment.rb (hunk 1 above)
Comment (AlexRS90) [reply app/models/appointment.rb:6]: Hello @codepantherr, thanks for your review.
The changes are done! 🤓

**Review by codepantha — APPROVED** [review]
Hi @AlexRS90 👋, 

# STATUS: APPROVED ✔️ ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AlexRS90** [comment]
Thanks @codepantherr 🚀

**Pull request merged**